content/en/docs/azure/deploy/install-kubeflow.md

```diff
@@ -67,7 +70,7 @@ Create a specifically defined cluster:
 
     az aks create -g <RESOURCE_GROUP_NAME> -n <NAME> -s <AGENT_SIZE> -c <AGENT_COUNT> -l <LOCATION> --generate-ssh-keys
```

Comment:
Line 71 (68): You need this code block to be surrounded by back ticks, for instance:

```diff
+     ```
    az aks create -g <RESOURCE_GROUP_NAME> -n <NAME> -s <AGENT_SIZE> -c <AGENT_COUNT> -l <LOCATION> --generate-ssh-keys
+     ```
```

Reply:
fixed